org.trustedqsl.Tqsl.metainfo.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 hamradio lu9dce -->
+<component type="desktop-application">
+ <id>org.trustedqsl.Tqsl</id>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>GPL-3.0</project_license>
+ <name>tqsl</name>
```

Comment:
```suggestion
 <name>Trusted QSL</name>
```

Reply:
`TrustedQSL` (without the space) in Fedora.